ui/scss/src/tinymce-ace.scss

```
@@ -146,6 +147,168 @@
+	.ace_search_form.ace_nomatch {
+		outline: 1px solid red;
+	}
+
+	.ace_search_field {
+		border-radius: 3px 0 0 3px;
+		background-color: white;
+		color: black;
+		border: 1px solid #cbcbcb;
+		border-right: 0 none;
+		outline: 0;
+		padding: 0;
```

Stylelint (17.14.0)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)